api_server/src/lib.rs

```diff
@@ -61,7 +61,7 @@ async fn query_graph(
     Extension(manager): Extension<Arc<RwLock<SchemaManager>>>,
 ) -> (StatusCode, Json<Value>) {
     if !manager.read().await.contains_key(&name) {
-        if let Ok(Some(schema)) = load_schema(&database_url, &name).await {
+        if let Ok(Some(schema)) = load_schema_wasm(&database_url, &name).await {
```

Comment:
You'll notice regular name changes such as this. Where "wasm" was added for clarity